data/tests/advisories/modules.yaml

```diff
@@ -32,6 +32,9 @@ data:
   license:
     module:
     - MIT
+  dependencies:
+    requires:
```

Comment:
```suggestion
  - requires:
```
I think here it has to have the `-` otherwise its not valid. The `recreate` script actually fails and the metadata are not updated (none of the checksums for modules.yaml have changed).

So interestingly this is not needed? Since the metadata in the repo are not updated it wasn't using this and the tests pass anyway.